third_party/xla/third_party/tsl/tsl/platform/threadpool.cc

```diff
@@ -21,6 +21,7 @@ limitations under the License.
 #include "unsupported/Eigen/CXX11/Tensor"  // from @eigen_archive
 #include "tsl/platform/blocking_counter.h"
 #include "tsl/platform/context.h"
+#include "tsl/platform/cpu_info.h"
```

Comment:
Just a quick heads up that I wrapped `#ifdef DNNL_AARCH64_USE_ACL` around this include to avoid having an unused include when `DNNL_AARCH64_USE_ACL` isn't defined.

Reply:
Ah yes that makes sense, thanks for adding that :)